src/tool_getparam.c

```diff
@@ -61,7 +61,7 @@
 } while(0)
 
 struct LongShort {
-  const char *letter; /* short name option */
+  const char *letter; /* internal short name option (see comment in aliases) */
```

Comment:
Not entirely true, as it can be the normal plain public short option **or** an internal one.

Reply:
But aren't they treated nearly the same?

Reply:
ACK